tests/integration/test_odbc_interaction/test.py

```diff
@@ -582,13 +622,13 @@ def test_postgres_odbc_hashed_dictionary_no_tty_pipe_overflow(started_cluster):
         "select dictGetString('postgres_odbc_hashed', 'column2', toUInt64(3))",
         "xxx",
     )
+    cursor.execute("truncate table clickhouse.test_table")
 
 
 def test_postgres_insert(started_cluster):
     skip_test_msan(node1)
 
     conn = get_postgres_conn(started_cluster)
-    conn.cursor().execute("truncate table clickhouse.test_table")
```

Comment:
`truncate` at the beginning of the test can be a bit better. Because `truncate` at the beginning makes the test less dependent on other tests, I mean if something fails it's more useful to see in the report what fails and not some other test.

Reply:
I do not agree. If test leaved trash after itself we should fix this one. If we failed to clean something it is also a failure of a test. Generally we have deterministic flow create, insert, delete, drop. There is no probabilistic reasons why we should add `IF EXISTS` etc.
I check that there is no leftovers after OK test. Usually I use `pytest --count 100` for it to run the same test multiple times on the same server.
When we clear before we actually hide issues of other tests.